renovate.json

```diff
@@ -2,5 +2,11 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["^@acme/"],
```

Comment:
This pattern would match @acme/expo and @acme/nextjs too and they are working now

Reply:
They aren't being used as dependencies anywhere so renovate isn't trying to find them anyway